pkg/descheduler/evictions/evictions.go

```diff
@@ -187,6 +152,66 @@ func evictPod(ctx context.Context, client clientset.Interface, pod *v1.Pod, poli
 	return err
 }
 
+type constraint func(pod *v1.Pod) error
+
+type evictable struct {
+	constraints []constraint
+}
+
+func (pe *PodEvictor) Evictable() *evictable {
```

Comment:
I think we should add some comments to these public functions.

Reply:
Done